StartEmailValidationProcess/handler.ts

```diff
@@ -159,7 +163,8 @@ export function StartEmailValidationProcess(
 
   const middlewaresWrap = withRequestMiddlewares(
     ContextMiddleware(),
-    FiscalCodeMiddleware
+    FiscalCodeMiddleware,
+    RequiredBodyPayloadMiddleware(EmailValidationProcessParams)
```

Comment:
Have you tested the endpoint without including any request body? I haven't done so myself, but it's a precautionary measure, considering that the backend won't send any data until deployment.

Reply:
with #305, we will release the SDK so the backend can send the data before merging this PR